README.md

```diff
@@ -181,6 +181,13 @@ When deploying for the first time, a Google Cloud Platform service account must
 
 When rotating credentials, the first step may be skipped.
 
+### Setting admin emails 
+
+Ensure to set admin account emails within AWS Secret Manager
```

Comment:
Maybe something like "Admin user emails are stored in AWS Secret Manager with the command:"

Reply:
Addressed